src/App/Documentation/core/Color/index.js

```diff
@@ -1,137 +1,149 @@
+            </div>
+            <div className="col-12 col-sm">
+                <ColorPreview name="brand-error" hex="#b24843" />
+            </div>
+        </div>
+        <div className="row">
+            <div className="col-12 col-sm">
+                <ColorPreview name="brand-error-light" hex="#fceeed" />
+            </div>
+            <div className="col-12 col-sm">
+                <ColorPreview name="brand-info" hex="#2c7a8f" />
+            </div>
+            <div className="col-12 col-sm">
+                <ColorPreview name="brand-info-light" hex="#f0f9fb" />
+            </div>
+        </div>
     </>
 );
 
 const Grayscale = () => (
     <>
         <h2 id="grayscale">Grayscale</h2>
-        <ComponentPreview language="html" showCasePanel>
-            <div className="row color-container">
-                <div className="col-12">
-                    <ColorPreview name="black" hex="#000000" color="#eaeaea" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="nearblack" hex="#111111" color="#eaeaea" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="gray-dark" hex="#262626" color="#eaeaea" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="gray-medium-dark" hex="#363636" color="#eaeaea" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="gray-medium" hex="#404040" color="#eaeaea" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="gray-medium-light" hex="#545454" color="#eaeaea" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="gray-light" hex="#666666" color="#eaeaea" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="gray-light-alt" hex="#999999" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="gray-bright" hex="#eaeaea" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="gray-bright-alt" hex="#bcbcbc" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="neargray" hex="#f9f9f9" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="offwhite" hex="#f4f4f4" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="nearwhite" hex="#f0f0f0" />
-                </div>
-                <div className="col-12">
-                    <ColorPreview name="white" hex="#ffffff" color="black" />
-                </div>
-            </div>
-        </ComponentPreview>
+        <div className="row mt-3">
```

Comment:
More of a general comment. We should start noticing where we use these vertical spacing classes, for future spacing adjustments.

Reply:
I'm not using `.showcase-panel` for colors, so there is no box with margin and padding separating it from the heading above. I added "margin-top" to give it more space. All of our documentation will have to be updated with new margin/padding if we stop using `.showcase-panel`.